README.md

```diff
@@ -1,62 +1,34 @@
 # A-Frame Core
 
-A DOM-based [Entity-Component System](https://en.wikipedia.org/wiki/Entity_component_system) to declaratively create 3D and VR worlds in the browser.
+![travis](https://travis-ci.org/aframevr/aframe-core.svg)
 
-> This library is specific to the core A-Frame library. For more information about the A-Frame project as a whole, head over to [the A-Frame site](https://aframe.io) or [the `aframe` repo](https://github.com/aframevr/aframe/).
+> There are currently plans for this repository to be the canonical [`aframe`](https://github.com/aframevr/aframe).
 
-## Entity-Component System
+A-Frame is a library for creating virtual reality web experiences.
 
-[Entity-Component System](https://en.wikipedia.org/wiki/Entity_component_system) is an architectural pattern commonly used in game engines such as Unity, PlayCanvas, an Unreal Engine 4+.
+- **Virtual Reality**: Drop in the library and have a virtual reality scene within a few lines of markup.
+- **Based on the DOM**: Manipulate with Javascript, use with your favorite libraries and frameworks.
```

Comment:
for "libraries and frameworks" perhaps link to https://github.com/aframevr/awesome-aframe/ ?